memcache_proxy/main/memcached2g.go

```diff
@@ -24,20 +23,20 @@ import (
 	"os"
 
 	"dtog"
+
 	gaeint "google.golang.org/appengine/notreallyinternal"
 )
 
 var (
 	bindingAddr = flag.String("binding_address", "localhost:11211",
-		"host:port the memcached daemon will listen on. For example 0.0.0.0:11211.")
+		"host:port the memcached daemon will listen on. For example, 0.0.0.0:11211")
 	application = flag.String("application", "", "app ID override")
 	module      = flag.String("module", "", "module override")
 	version     = flag.String("version", "", "major version override")
 	instance    = flag.String("instance", "", "instance override")
 	apiHost     = flag.String("api_host", "appengine.googleapis.internal", "for dev, set to the 'localhost'")
 	apiPort     = flag.String("api_port", "10001",
-		"for dev, set to the <port>' as reported by dev_appserver: 'Starting API "+
-			"server at: http://localhost:<port>'")
+		"for dev, set to the <port>' as reported by dev_appserver: 'Starting API server at: http://localhost:<port>'")
```

Comment:
cleanup extra '

It also seems a little inconsistent to have this host and port split.